### PR TITLE
Add whatsapp bot

### LINKED ISSUE #451: Whatsapp Bot
# Aim

- What is the objective of the Script
Whatsapp Bot
# Details
- What the features will your script have
The scripts will do the Following Tasks
To automatically send a message to a group of people individually.
To schedule your message
Voice Controlled Bot

### PR DESCRIPTION
**Pull request by muthuannamalai12**
# Description
I have added a Whatsapp Bot which performs the following task
To automatically send a message to a group of people individually.
To schedule your message
Voice Controlled Bot
Please include a summary of the change and which issue is fixed. List any dependencies that are required for this change.

## Fixes #451 

Replace `issue_no` with the issue number which is fixed in this PR

## Have you read the [Contributing Guidelines on Pull Requests](https://github.com/avinashkranjan/Amazing-Python-Scripts/blob/master/CONTRIBUTING.md)?

- [x] Yes
- [ ] No

## Type of change

_Please delete options that are not relevant._

- [x] New feature (non-breaking change which adds functionality)
- [x] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [x] This change requires a documentation update
- [x] Documentation Update

## Checklist:

- [x] My code follows the style guidelines(Clean Code) of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have created a helpful and easy to understand `README.md`
- [x] My documentation follows [`Template for README.md`](https://github.com/avinashkranjan/Amazing-Python-Scripts/blob/master/Template%20for%20README.md)
- [x] My changes generate no new warnings
- [x] I have added tests/screenshots(if any) that prove my fix is effective or that my feature works.

### CODE REVIEW

**Comment by kaustubhgupta** [comment]
@muthuannamalai12 kindly check the code so that all the test pass and after that your PR will be reviewed

**Comment by kaustubhgupta** [comment]
To pass the Lint tests, Kindly do the following:
- download autopep8 using pip
- Go to your project directory.
- Run the command: 
    ```bash 
       autopep8 -i -r --verbose ./
    ```

This should help you pass the test

**Comment by kaustubhgupta** [comment]
updates @muthuannamalai12 ?

**Comment by muthuannamalai12** [comment]
> updates @muthuannamalai12 ?

I will complete it within tomorrow

**Comment by kaustubhgupta** [comment]
@muthuannamalai12 and the insta bot?

**Comment by muthuannamalai12** [comment]
> @muthuannamalai12 and the insta bot?

I will complete Insta Bot also within tomorrow.

**Comment by muthuannamalai12** [comment]
> To pass the Lint tests, Kindly do the following:
> 
> * download autopep8 using pip
> * Go to your project directory.
> * Run the command:
>   ```shell
>      autopep8 -i -r --verbose ./
>   ```
> 
> This should help you pass the test

I have done it still the error is present

**Comment by muthuannamalai12** [comment]
image

After I apply the command it shows 0 error is present

File: whatsapp-bot/whatsapp-bot.py [hunk 1]
```diff
@@ -0,0 +1,92 @@
+from datetime import datetime
+from time import sleep
+
+from pandas import read_csv
+from selenium import webdriver
+from Voice_control_Module.py import *
```
Comment (kaustubhgupta) [inline whatsapp-bot/whatsapp-bot.py:6]:
```suggestion
from Voice_control_Module import *
```

**Review by kaustubhgupta — CHANGES_REQUESTED** [review]
I got your error and you will be shocked what was your mistake:

**Comment by muthuannamalai12** [comment]
I have updated it @kaustubhgupta

**Comment by muthuannamalai12** [comment]
I have updated @kaustubhgupta. Kindly review again

**Review by kaustubhgupta — COMMENTED** [review]
I am not able to run this bot locally, @antrikshmisri @santushtisharma10 can you confirm if this working or not?
image

**Comment by kaustubhgupta** [comment]
@muthuannamalai12 you haven't responded about this in 2 days. I can give you the last 6 Hrs to fix this after that this PR will be closed.

**Review by kaustubhgupta — CHANGES_REQUESTED** [review]
Did you ran this script? There are a lot of bugs here:
- There is not purpose of CSV file as you have not used the loadContacts function anywhere
- You are taking the user input as user_name but in the send message function, it is phone number list.

**Comment by kaustubhgupta** [comment]
@muthuannamalai12 I can give you the last 6 Hrs to fix these issues after that this PR will be closed.

**Comment by muthuannamalai12** [comment]
> @muthuannamalai12 I can give you the last 6 Hrs to fix these issues after that this PR will be closed.

I have been trying to correct it I will do my level best If I cannot make it I will close the Pull Request

**Comment by kaustubhgupta** [comment]
> > @muthuannamalai12 I can give you the last 6 Hrs to fix these issues after that this PR will be closed.
> 
> I have been trying to correct it I will do my level best If I cannot make it I will close the Pull Request

This script is not even running for one case, didn't you tested this before pushing the code?

**Comment by kaustubhgupta** [comment]
@muthuannamalai12 Please update the final status?

**Comment by muthuannamalai12** [comment]
> @muthuannamalai12 Please update the final status?

I tried my best but cannot solve the bug

**Comment by kaustubhgupta** [comment]
@muthuannamalai12 Okay no issues, you can work on the other two newly assigned issues. These two (Whatsapp bot and insta bot) will be assigned to someone else

**Pull request closed without merging**